### DISCUSSION TITLE
Need Help in running magma LTE AGW

### DISCUSSION THREAD

**Discussion by Flextride** [magma/magma#12865] [Q&A]
Hello,

I am following quick start guide : https://magma.github.io/magma/docs/basics/quick_start_guide. and I am able to run orchestrator as mentioned in this guide. But while running Magma  service (sudo service magma@magmad restart) I get these errors [I am using 1.6 tag]:

`May 29 17:41:21 magma-dev-focal systemd-udevd[2656016]: ethtool: autonegotiation is unset or enabled, the speed and duplex are not writable.
May 29 17:41:21 magma-dev-focal systemd-udevd[2656016]: envoy_cntr_ns: Could not generate persistent MAC: No data available
May 29 17:41:21 magma-dev-focal systemd-networkd[315224]: envoy_cntr: Lost carrier
May 29 17:41:21 magma-dev-focal systemd-udevd[2656016]: ethtool: autonegotiation is unset or enabled, the speed and duplex are not writable.
May 29 17:41:21 magma-dev-focal systemd-udevd[2656016]: envoy_cntr_ns: Could not generate persistent MAC: No data available
May 29 17:41:22 magma-dev-focal control_proxy[1115478]: 2022-05-29T17:41:22.329Z [127.0.0.1 -> metricsd-controller.magma.test,8443] "POST /magma.orc8r.MetricsController/Collect HTTP/2" 502 129bytes 0.000s
May 29 17:41:22 magma-dev-focal magmad[1115221]: ERROR:root:Metrics upload error for service enodebd (chunk 0)! [StatusCode.UNKNOWN] Stream removed
May 29 17:41:22 magma-dev-focal control_proxy[1115478]: 2022-05-29T17:41:22.329Z [127.0.0.1 -> metricsd-controller.magma.test,8443] "POST /magma.orc8r.MetricsController/Collect HTTP/2" 502 129bytes 0.000s
May 29 17:41:22 magma-dev-focal magmad[1115221]: ERROR:root:Metrics upload error for service pipelined (chunk 0)! [StatusCode.UNKNOWN] Stream removed
May 29 17:41:22 magma-dev-focal systemd-networkd[315224]: envoy_cntr: Link DOWN
May 29 17:41:23 magma-dev-focal magmad[1115221]: E0529 17:41:23.176120930 1115247 ssl_transport_security.cc:1495] Handshake failed with fatal error SSL_ERROR_SSL: error:0400006b:RSA routines:OPENSSL_internal:BLOCK_TYPE_IS_NOT_01.
May 29 17:41:24 magma-dev-focal systemd[1]: run-netns-envoy_ns1.mount: Succeeded.
May 29 17:41:24 magma-dev-focal systemd[2655602]: run-netns-envoy_ns1.mount: Succeeded.
May 29 17:41:24 magma-dev-focal systemd[1]: magma@envoy_controller.service: Failed with result 'exit-code'.
May 29 17:41:24 magma-dev-focal control_proxy[1115478]: 2022-05-29T17:41:24.618Z [127.0.0.1 -> metricsd-controller.magma.test,8443] "POST /magma.orc8r.MetricsController/Collect HTTP/2" 502 129bytes 0.000s
May 29 17:41:24 magma-dev-focal magmad[1115221]: ERROR:root:Metrics upload error for service state (chunk 0)! [StatusCode.UNKNOWN] Stream removed
May 29 17:41:24 magma-dev-focal smsd[1115386]: INFO:root:mme non_eps_service_config is not SMS_ORC8R, sleeping.
May 29 17:41:25 magma-dev-focal control_proxy[1115478]: 2022-05-29T17:41:25.093Z [127.0.0.1 -> metricsd-controller.magma.test,8443] "POST /magma.orc8r.MetricsController/Collect HTTP/2" 502 129bytes 0.000s
May 29 17:41:25 magma-dev-focal magmad[1115221]: ERROR:root:Metrics upload error for service sessiond (chunk 0)! [StatusCode.UNKNOWN] Stream removed
May 29 17:41:26 magma-dev-focal magmad[1115221]: ERROR:root:GetServiceInfo Error for envoy_controller! [StatusCode.UNAVAILABLE] failed to connect to all addresses
May 29 17:41:29 magma-dev-focal systemd[1]: magma@envoy_controller.service: Scheduled restart job, restart counter is at 1108.
May 29 17:41:29 magma-dev-focal systemd[1]: Stopped Magma envoy controller service.
May 29 17:41:29 magma-dev-focal systemd[1]: Starting Magma envoy controller service...
May 29 17:41:30 magma-dev-focal networkd-dispatcher[393701]: WARNING:Unknown index 25531 seen, reloading interface list
May 29 17:41:30 magma-dev-focal systemd-udevd[2656089]: ethtool: autonegotiation is unset or enabled, the speed and duplex are not writable.
May 29 17:41:30 magma-dev-focal systemd-udevd[2656089]: Using default interface naming scheme 'v245'.
May 29 17:41:30 magma-dev-focal systemd-udevd[2656089]: envoy_cntr_ns: Could not generate persistent MAC: No data available
May 29 17:41:30 magma-dev-focal systemd-udevd[2656094]: ethtool: autonegotiation is unset or enabled, the speed and duplex are not writable.
May 29 17:41:30 magma-dev-focal systemd-udevd[2656094]: Using default interface naming scheme 'v245'.
May 29 17:41:30 magma-dev-focal systemd-udevd[2656094]: envoy_cntr: Could not generate persistent MAC: No data available
May 29 17:41:30 magma-dev-focal systemd-networkd[315224]: envoy_cntr: Link UP
May 29 17:41:31 magma-dev-focal kernel: [513337.084955] IPv6: ADDRCONF(NETDEV_CHANGE): envoy_cntr_ns: link becomes ready
May 29 17:41:31 magma-dev-focal kernel: [513337.085078] IPv6: ADDRCONF(NETDEV_CHANGE): envoy_cntr: link becomes ready
May 29 17:41:31 magma-dev-focal systemd-networkd[315224]: envoy_cntr: Gained carrier
May 29 17:41:31 magma-dev-focal smsd[1115386]: INFO:root:mme non_eps_service_config is not SMS_ORC8R, sleeping.
May 29 17:41:31 magma-dev-focal kernel: [513337.257548] IPv6: ADDRCONF(NETDEV_CHANGE): proxy_port_ns: link becomes ready
May 29 17:41:31 magma-dev-focal systemd-networkd[315224]: proxy_port: Gained carrier
^C
vagrant@magma-dev-focal:~$ sudo tail -f /var/log/syslog
vagrant@magma-dev-focal:~$ sudo tail -f /var/log/syslog
May 29 17:49:56 magma-dev-focal smsd[1115386]: INFO:root:mme non_eps_service_config is not SMS_ORC8R, sleeping.
May 29 17:49:56 magma-dev-focal systemd-networkd[315224]: envoy_cntr: Link DOWN
May 29 17:49:57 magma-dev-focal systemd[1]: run-netns-envoy_ns1.mount: Succeeded.
May 29 17:49:57 magma-dev-focal systemd[2655602]: run-netns-envoy_ns1.mount: Succeeded.
May 29 17:49:57 magma-dev-focal systemd[1]: magma@envoy_controller.service: Failed with result 'exit-code'.
May 29 17:49:58 magma-dev-focal magmad[1115221]: ERROR:root:GetServiceInfo Error for envoy_controller! [StatusCode.UNAVAILABLE] failed to connect to all addresses
May 29 17:50:02 magma-dev-focal smsd[1115386]: INFO:root:mme non_eps_service_config is not SMS_ORC8R, sleeping.
May 29 17:50:02 magma-dev-focal systemd[1]: magma@envoy_controller.service: Scheduled restart job, restart counter is at 1126.
May 29 17:50:02 magma-dev-focal systemd[1]: Stopped Magma envoy controller service.
May 29 17:50:02 magma-dev-focal systemd[1]: Starting Magma envoy controller service...
May 29 17:50:03 magma-dev-focal networkd-dispatcher[393701]: WARNING:Unknown index 25567 seen, reloading interface list
May 29 17:50:03 magma-dev-focal systemd-udevd[2659012]: ethtool: autonegotiation is unset or enabled, the speed and duplex are not writable.
May 29 17:50:03 magma-dev-focal systemd-udevd[2659012]: Using default interface naming scheme 'v245'.
May 29 17:50:03 magma-dev-focal systemd-udevd[2659012]: envoy_cntr: Could not generate persistent MAC: No data available
May 29 17:50:03 magma-dev-focal systemd-udevd[2659009]: ethtool: autonegotiation is unset or enabled, the speed and duplex are not writable.
May 29 17:50:03 magma-dev-focal systemd-udevd[2659009]: Using default interface naming scheme 'v245'.
May 29 17:50:03 magma-dev-focal systemd-udevd[2659009]: envoy_cntr_ns: Could not generate persistent MAC: No data available
May 29 17:50:04 magma-dev-focal systemd-networkd[315224]: envoy_cntr: Link UP
May 29 17:50:05 magma-dev-focal kernel: [513850.915986] IPv6: ADDRCONF(NETDEV_CHANGE): envoy_cntr_ns: link becomes ready
May 29 17:50:05 magma-dev-focal kernel: [513850.916114] IPv6: ADDRCONF(NETDEV_CHANGE): envoy_cntr: link becomes ready
May 29 17:50:05 magma-dev-focal kernel: [513850.916281] IPv6: ADDRCONF(NETDEV_CHANGE): proxy_port_ns: link becomes ready
May 29 17:50:05 magma-dev-focal systemd-networkd[315224]: envoy_cntr: Gained carrier
May 29 17:50:05 magma-dev-focal systemd-networkd[315224]: proxy_port: Gained carrier
May 29 17:50:06 magma-dev-focal magmad[1115221]: ERROR:root:GetOperationalStates Error for envoy_controller! [StatusCode.UNAVAILABLE] failed to connect to all addresses
May 29 17:50:06 magma-dev-focal magmad[1115221]: E0529 17:50:06.598050918 1115221 fork_posix.cc:76]           Other threads are currently calling into gRPC, skipping fork() handlers
May 29 17:50:07 magma-dev-focal envoy_controller[2659074]: net.ipv4.conf.all.rp_filter = 0
May 29 17:50:07 magma-dev-focal control_proxy[1115478]: 2022-05-29T17:50:07.347Z [127.0.0.1 -> state-controller.magma.test,8443] "POST /magma.orc8r.StateService/ReportStates HTTP/2" 502 129bytes 0.001s
May 29 17:50:07 magma-dev-focal envoy_controller[2659075]: net.ipv4.conf.all.route_localnet = 1
May 29 17:50:07 magma-dev-focal magmad[1115221]: ERROR:root:Checkin Error! Failed to report states. [StatusCode.UNKNOWN] Stream removed
May 29 17:50:07 magma-dev-focal magmad[1115221]: INFO:root:StateReporting (Checkin) failure threshold met, remediating...
May 29 17:50:07 magma-dev-focal systemd[1]: Started Magma envoy controller service.
May 29 17:50:07 magma-dev-focal systemd[2659077]: magma@envoy_controller.service: Failed to execute command: No such file or directory
May 29 17:50:07 magma-dev-focal systemd-networkd[315224]: envoy_cntr: Gained IPv6LL
May 29 17:50:07 magma-dev-focal systemd[2659077]: magma@envoy_controller.service: Failed at step EXEC spawning /usr/local/bin/envoy_controller: No such file or directory
May 29 17:50:07 magma-dev-focal systemd[1]: magma@envoy_controller.service: Main process exited, code=exited, status=203/EXEC
May 29 17:50:07 magma-dev-focal magmad[1115221]: ERROR:root:Error making SSL connection: No such file or directory, [Errno 2] No such file or directory
May 29 17:50:07 magma-dev-focal magmad[1115221]: ERROR:root:Bootstrapping due to invalid cert
May 29 17:50:07 magma-dev-focal magmad[1115221]: E0529 17:50:07.955632112 1115247 ssl_transport_security.cc:1495] Handshake failed with fatal error SSL_ERROR_SSL: error:0400006b:RSA routines:OPENSSL_internal:BLOCK_TYPE_IS_NOT_01.
May 29 17:50:07 magma-dev-focal magmad[1115221]: ERROR:root:GetChallenge error! [StatusCode.UNAVAILABLE] failed to connect to all addresses
May 29 17:50:07 magma-dev-focal magmad[1115221]: INFO:root:Retrying bootstrap in 30 seconds
May 29 17:50:08 magma-dev-focal magmad[1115221]: E0529 17:50:08.563805764 1115221 fork_posix.cc:76]           Other threads are currently calling into gRPC, skipping fork() handlers
May 29 17:50:08 magma-dev-focal magmad[1115221]: E0529 17:50:08.922934204 1115247 ssl_transport_security.cc:1495] Handshake failed with fatal error SSL_ERROR_SSL: error:0400006b:RSA routines:OPENSSL_internal:BLOCK_TYPE_IS_NOT_01.
May 29 17:50:09 magma-dev-focal smsd[1115386]: INFO:root:mme non_eps_service_config is not SMS_ORC8R, sleeping.
May 29 17:50:10 magma-dev-focal magmad[1115221]: E0529 17:50:10.320640927 1115221 fork_posix.cc:76]           Other threads are currently calling into gRPC, skipping fork() handlers
May 29 17:50:10 magma-dev-focal magmad[1115221]: E0529 17:50:10.688142933 1115247 ssl_transport_security.cc:1495] Handshake failed with fatal error SSL_ERROR_SSL: error:0400006b:RSA routines:OPENSSL_internal:BLOCK_TYPE_IS_NOT_01.
May 29 17:50:11 magma-dev-focal magmad[1115221]: /home/vagrant/build/python/lib/python3.8/site-packages/psutil/_pslinux.py:1226: RuntimeWarning: ignoring FileNotFoundError(2, 'No such file or directory') for file '/sys/class/hwmon/hwmon0/temp1_input'
May 29 17:50:11 magma-dev-focal magmad[1115221]:   warnings.warn("ignoring %r for file %r" % (err, path),
May 29 17:50:11 magma-dev-focal magmad[1115221]: /home/vagrant/build/python/lib/python3.8/site-packages/psutil/_pslinux.py:1226: RuntimeWarning: ignoring FileNotFoundError(2, 'No such file or directory') for file '/sys/class/hwmon/hwmon0/temp2_input'
May 29 17:50:11 magma-dev-focal magmad[1115221]:   warnings.warn("ignoring %r for file %r" % (err, path),
May 29 17:50:11 magma-dev-focal magmad[1115221]: ERROR:root:GetServiceInfo Error for envoy_controller! [StatusCode.UNAVAILABLE] failed to connect to all addresses
May 29 17:50:12 magma-dev-focal magmad[1115221]: E0529 17:50:12.736004816 1115247 ssl_transport_security.cc:1495] Handshake failed with fatal error SSL_ERROR_SSL: error:0400006b:RSA routines:OPENSSL_internal:BLOCK_TYPE_IS_NOT_01.
^VMay 29 17:50:13 magma-dev-focal control_proxy[1115478]: 2022-05-29T17:50:13.239Z [127.0.0.1 -> streamer-controller.magma.test,8443] "POST /magma.orc8r.Streamer/GetUpdates HTTP/2" 502 129bytes 0.000s
May 29 17:50:13 magma-dev-focal policydb[1115371]: ERROR:root:Error! Streaming from the cloud failed! [StatusCode.UNKNOWN] Stream removed
May 29 17:50:13 magma-dev-focal control_proxy[1115478]: 2022-05-29T17:50:13.239Z [127.0.0.1 -> streamer-controller.magma.test,8443] "POST /magma.orc8r.Streamer/GetUpdates HTTP/2" 502 129bytes 0.000s
May 29 17:50:13 magma-dev-focal policydb[1115371]: ERROR:root:Error! Streaming from the cloud failed! [StatusCode.UNKNOWN] Stream removed
May 29 17:50:13 magma-dev-focal control_proxy[1115478]: 2022-05-29T17:50:13.239Z [127.0.0.1 -> streamer-controller.magma.test,8443] "POST /magma.orc8r.Streamer/GetUpdates HTTP/2" 502 129bytes 0.000s
May 29 17:50:13 magma-dev-focal policydb[1115371]: ERROR:root:Error! Streaming from the cloud failed! [StatusCode.UNKNOWN] Stream removed
^V^V^VMay 29 17:50:15 magma-dev-focal smsd[1115386]: INFO:root:mme non_eps_service_config is not SMS_ORC8R, sleeping.`

I tried figuring out these errors, but no luck. Can somebody please give some pointers, what is broken in my system? what to look into ? . 

Thanks
pankaj

**Comment by sebathomas** [comment]
You should probably follow the quick start guide for release 1.6: https://magma.github.io/magma/docs/1.6.X/basics/quick_start_guide
(Not sure if there are any major differences though.)

Have you set up the connection between agw and orc8r via `fab -f dev_tools.py register_vm`?

**Reply by Flextride** [reply]
Yes I did run "fab -f dev_tools.py register_vm"
